Docs/source/Model.rst

```diff
@@ -741,52 +740,63 @@ projection equation
 
 .. math::
 
-    D^{MAC} \frac{1}{\rho} \delta e^{\ell} = D^{MAC} \delta U^{ADV,\ell}
+    D^{MAC} \frac{1}{\rho} \delta e^{\ell} = D^{MAC} \delta U^{ADV,\ell} + \delta_{\chi}^{\ell}
 
+where :math:`\delta_{\chi}^{\ell}` is incremented iteratively to enfoce the final state to satisfy the EOS
 and compute the correction velocity
 
 .. math::
 
     U^{ADV,\ell,corr} = -\frac{1}{\rho} G^{MAC} \delta e^{\ell}
 
-which is the increment of velocity to carry advection fluxes required to correct the errors made by advancing the
+which is the increment of velocity required to carry advection fluxes needed to correct the errors made by advancing the
 coarse state with the wrong velocities.
 
-The second part of the mismatch arises because the advective and diffusive fluxes on the coarse grid were computed without explicitly accounting for the fine grid, while on the fine grid the fluxes were computed using coarse-grid Dirichlet boundary data.  We define the flux discrepancies 
+The second part of the mismatch arises because the advective and diffusive fluxes on the coarse grid were computed 
+without explicitly accounting for the fine grid, while on the fine grid the fluxes were computed using coarse-grid 
+Dirichlet boundary data.  We define the flux discrepancies on the coarser level :math:`\ell` of the pair of levels considered:
```

Comment:
Mention that formally, the synchronization is a bilevel correction that should be computed as a sequence of two-level solves.  However, this would lead to the same amount of work that was required to create the original (pre-sync) data.  We assume the the corrections computed for the sync are smooth enough to be well-represented by an increment computed on the coarser of the two levels, and interpolated to the finer grids.

Reply:
Also, the sync is computed using the "new-time" transport coefficients, as last computed in the level advance iterations - I.e., we do not update them to account for state changes in the sync. However, the states themselves are computed thermodynamically consistent.

Reply:
Ok done too but you might want to check the text